lib/resque/scheduler/scheduling_extensions.rb

```diff
@@ -84,8 +84,16 @@ def schedule
       end
 
       # reloads the schedule from redis
-      def reload_schedule!
+      def reload_schedule!(verify_with: nil, max_retries: 7)
+        retries = 0
+
         @schedule = all_schedules
+        while verify_with && retries < max_retries &&
+              !(@schedule && @schedule.eql?(verify_with))
+          @schedule = all_schedules
+          retries += 1
+          sleep 2**(retries - rand + 1) * 0.1 unless @schedule
```

Comment:
so that two or more processes don't back off in a uniform manner, again colliding with each other